examples/Demo/FluentUI.Demo.Client/Layout/DemoNavMenu.razor.css

```diff
@@ -6,7 +6,6 @@ ul {
 
 li {
   font-weight: 600;
-  padding: 8px 10px;
```

Comment:
Why did you remove this style? The result is not the same :-(

Before
![Image](https://github.com/user-attachments/assets/c8ec4af2-b4d7-41cd-8760-c23760a67cd5)

After
![Image](https://github.com/user-attachments/assets/689aeb4a-b9d5-473e-a307-31a4a22d33c6)